### PR TITLE
solution

### PR DESCRIPTION
**Pull request by lefteris-L**

### CODE REVIEW

**Review by ysrckr — APPROVED** [review]
Looks good 🤟🏻

File: src/arrayMethodReduce.js [hunk 1]
```diff
@@ -5,7 +5,14 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let result = arguments.length < 2 ? this[0] : initialValue;
+    let i = arguments.length < 2 ? 1 : 0;
```
Comment (naziamoff) [inline src/arrayMethodReduce.js:9]: probably startIndex will sound better here?

Comment (naziamoff) [inline src/arrayMethodReduce.js:9]: ternaries should be multi-lined

**Review by naziamoff — APPROVED** [review]
Well done, but consider my comments pls

File: src/arrayMethodReduce.js (hunk 1 above)
Comment (lefteris-L) [reply src/arrayMethodReduce.js:9]: I see, so, if `i` is declared outside the loop, it should have a clearer name, right?

**Pull request closed without merging**